app/templates/fragments/navigation_bar.html

```diff
@@ -42,8 +42,8 @@
             >
               Help
             </a>
-            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <a class="dropdown-item" href="/#/">Documentation</a>
+            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+              <a class="dropdown-item" href="https://github.com/CONP-PCNO/conp-portal">Documentation</a>
```

Comment:
https://github.com/CONP-PCNO/conp-portal/wiki
```suggestion
              <a class="dropdown-item" href="https://github.com/CONP-PCNO/conp-portal/wiki">Documentation</a>
```